db/migrate/20230326064936_create_chhand_types.rb

```diff
@@ -0,0 +1,9 @@
+class CreateChhandTypes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :chhand_types do |t|
+      t.string :chhand_name
```

Comment:
Nice job! 

I was under the impresion that it the word `name` was reserved keyword for databases. 

Turns out, it's not. 

We can probably simplify this by naming it `name`. 

That way, we can do:

`@chhand.chhand_type.name` 

instead of:

`@chhand.chhand_type.chhand_name`.

Reply:
Another thing that we can do is... this should by NON NULL.